Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -26,22 +28,59 @@
     solution: food
     useSound:
       path: /Audio/Items/eating_1.ogg
+  - type: Damageable
+    damageContainer: Inorganic
+  - type: Spillable
+    solution: food
+  - type: TrashOnEmpty
+    solution: food
+
+- type: entity
+  abstract: true
+  parent: ReagentContainerBase
+  id: ReagentPacketBase
+  components:
   - type: Openable
     sound:
       collection: packetOpenSounds
-  - type: Spillable
-    solution: food
+  # packet can be broken open
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 2
+      behaviors:
+      - !type:PlaySoundBehavior
+        sound:
+          collection: desecration
+      - !type:SpillBehavior
+        solution: food
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+  # packet contents can be splashed when open
   - type: MeleeWeapon
     soundNoDamage:
       path: "/Audio/Effects/Fluids/splat.ogg"
     damage:
       types:
         Blunt: 0
-  - type: TrashOnEmpty
-    solution: food
 
 - type: entity
-  parent: ReagentContainerBase
+  abstract: true
+  id: ItemHeftyBase
+  description: hit hard ye
```

Comment:
Yeet this and the rest is guzzy

Reply:
but 4/5 things use it

Reply:
I mean the desc, not the prot

Reply:
bang stick gibber [DEBUG] reference fake